src/foundry/foundry.js/hooks.d.ts

```diff
@@ -675,11 +685,12 @@ declare global {
     type HoverPlaceableObject<P extends PlaceableObject = PlaceableObject> = (object: P, hover: boolean) => unknown;
 
     /**
-     * @remarks This is called after copying {@link PlaceableObject}s in a copy-paste action, but before embedding them
-     * into the {@link PlaceablesLayer}.
-     * @param copied   - the originally copied PlaceableObjects
-     * @param toCreate - the pasted copies with new coordinates
-     * @param P        - the type of the PlaceableObject
+     * A hook event that fires when any PlaceableObject is pasted onto the
+     * Scene. Substitute the PlaceableObject name in the hook event to target a
+     * specific PlaceableObject type, for example "pasteToken".
+     * @param copied     - The PlaceableObjects that were copied
+     * @param createData - The new objects that will be added to the Scene
```

Comment:
The parameters in the function definition should be renamed according to the documentation.